quality/postman/tests_preconditions.postman_collection.json

```diff
@@ -149,6 +149,13 @@
 								"uac",
 								"1",
 								"api-key"
+							],
+							"query": [
+								{
+									"key": "key",
+									"value": "abcde12345abcde12345abcde12345abcde12345abcde123",
```

Comment:
is this default value really needed to be specified?

Reply:
No, removed.